packages/lit-xml/readme.md

```diff
@@ -1,5 +1,9 @@
 # Lit XML 🔥
 
+<p align="center">
```

Comment:
Align center? Sure this is correct html?

Reply:
Yup, here is Bill Murray to explain it: https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d#center-alignment-example